### ISSUE TITLE
Nice library. But...

### ISSUE THREAD

**Issue by JTrotta** [looterwar/NavigationLib#3]
...flickering, during page navigation is fulsome.
Any idea to reduce it?

**Comment by jpreece6** [comment]
You could try setting double buffered to true on the parent form and effected page. How many controls do you have on the page? More controls will probably increase flickering.

**Comment by JTrotta** [comment]
Nope! Double buffered does not work. The best solution is to override this method into the host form:
`        /// <summary>
        /// Avoid flickering during add/remove controls
        /// </summary>
        protected override CreateParams CreateParams
        {
            get
            {
                CreateParams cp = base.CreateParams;
                cp.ExStyle |= 0x02000000;  // Turn on WS_EX_COMPOSITED
                return cp;
            }
        } `
